lib/MusicBrainz/Server/Edit/Work/RelatedEntities.pm

```diff
@@ -13,12 +13,29 @@ around '_build_related_entities' => sub
         $self->c->model('Work')->get_by_ids($self->work_ids)
     };
 
+    my ($recordings, $recording_hits) = $self->c->model('Recording')->find_by_works($self->work_ids);
+    my @recordings = @$recordings;
+
+    my @recording_ids = map { $_->id } @recordings;
+
+    my @releases;
+    if (@recording_ids) {
+        my ($releases, $release_hits) = $self->c->model('Release')->find_by_recording(\@recording_ids);
+        @releases = @$releases;
+    }
```

Comment:
Just the `my ($releases, $release_hits) = ...` line seems necessary, since `find_by_recording` already returns early if the list is empty. Avoiding the function call by checking `if (@recording_ids)` is nice, but on the other hand it'd be simpler if we didn't have to declare `my @releases` and copy the result.